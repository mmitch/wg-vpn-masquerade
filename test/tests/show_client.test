../wg-conf client host1
