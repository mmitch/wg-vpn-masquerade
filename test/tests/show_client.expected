[Interface]
Address = 10.10.10.11/24
PrivateKey = PRIVATE-KEY-1
DNS = 10.10.10.10

[Peer]
PublicKey = PUBLIC-SERVER-KEY
AllowedIPs = 0.0.0.0/0, ::/0
Endpoint = SERVER-IP:51820
